tests/python/pants_test/backend/python/interpreter_selection_utils.py

```diff
@@ -46,6 +48,8 @@ def python_interpreter_path(version):
   :returns: the normalized path to the interpreter binary if found; otherwise `None`
   :rtype: string
   """
+  if PY2:
```

Comment:
I'd slightly prefer `except (subprocess.CalledProcessError, FileNotFoundError if PY3 else IOError):`, because avoiding weird assignments that look like they kind of shadow globals is weird, but this works fine too :)